apigee/edge_client.go

```diff
@@ -29,18 +29,22 @@ import (
 	"net/url"
 	"os"
 	"path"
+	"strings"
 
 	"github.com/bgentry/go-netrc/netrc"
 )
 
 const (
-	libraryVersion = "0.1.0"
-	defaultBaseURL = "https://api.enterprise.apigee.com/"
-	userAgent      = "go-apigee-edge/" + libraryVersion
-	appJSON        = "application/json"
-	octetStream    = "application/octet-stream"
+	libraryVersion  = "0.1.0"
+	defaultBaseURL  = "https://api.enterprise.apigee.com"
+	basicAuthHeader = "Basic ZWRnZWNsaTplZGdlY2xpc2VjcmV0"
```

Comment:
What is this?

Reply:
This is the required hard-coded basic auth to fetch the access token. It's a centralized credential.
Ref: https://docs.apigee.com/api-platform/system-administration/management-api-tokens#get-a-new-access-token

Reply:
Oh, that's wild. Ok, thanks.

Reply:
Agreed.